src/components/Sidebar.tsx

```diff
@@ -1,4 +1,63 @@
-import { Box } from '@mui/material';
-import React from 'react';
+import { Box, Divider, Grid, IconButton, List } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { SidebarMenuItem } from './SidebarMenuItem';
+import { ISidebarMenuItem } from 'src/models/SidebarMenuItem';
+import MenuIcon from '@mui/icons-material/Menu';
+import React, { useState } from 'react';
+import ReceiptLongIcon from '@mui/icons-material/ReceiptLong';
+import Tooltip from '@mui/material/Tooltip';
+import ViewSidebarIcon from '@mui/icons-material/ViewSidebar';
 
-export const Sidebar = () => <Box>Sidebar</Box>;
+export const Sidebar = () => {
+  const { t } = useTranslation();
+
+  const [collapsed, setCollapsed] = useState<boolean>(false);
```

Comment:
Could you move this in an external and reusable hook (it's a toggle) so that we can write unit tests easily?